### ISSUE TITLE
Update WSF Ambition 7 numbers

### ISSUE THREAD

**Issue by nsorros** [wellcometrust/policytool#132]
> These documents cited **130,139 articles in total. 1.4% of these (n = 1778)** were Wellcome research. However, some individual documents contained a much higher percentage of Wellcome research. For example, in the 2012 WHO technical report ‘Research priorities for helminth infections: technical report of the TDR disease reference group on helminth infections’ there were **539 references cited, 11.7% (n=63)** were Wellcome research.

Marta is asking an update on those numbers and I am not 100% sure how to replicate them. 
When I run this into the db
```
select
	count(distinct file_hash)
from document;
```
I get **5,228,096** which is far from the previous number. Did we filter the references somehow before reporting?

Also how did we calculate the last number?

**Comment by lizgzil** [comment]
btw I assume this has an error in it, because 5,228,096 is the number of:
```
select
	count(distinct id)
from document;
```
not distinct file_hash which is 5124.

The only things I can think is
1. the number has genuinely increased loads since other parts of the algo have improved (not sure what data I got this 130,139 number on).
2. at some point we did have duplicate policy documents in the db (see https://github.com/wellcometrust/policytool/issues/48 and https://github.com/wellcometrust/policytool/issues/104), so there is a chance of the same doc and reference repeating in the db

I did:
```
select
	count(distinct concat(file_hash, '-', title,'-', author,'-', journal,'-', pub_year,'-', issue,'-', volume))
from document;
```
and got 1,131,488 counts. Which might mean issue 104 is still at play (duplicated policy docs).

Thus I think there is still an error somewhere with duplicated data, and thus each row of `document` not truly being a unique reference from a policy document.
